skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkiaLayer.ios.kt

```diff
@@ -3,13 +3,17 @@ package org.jetbrains.skiko
 import kotlinx.cinterop.useContents
 import org.jetbrains.skia.Canvas
 import org.jetbrains.skia.PixelGeometry
-import org.jetbrains.skiko.context.MetalContextHandler
-import org.jetbrains.skiko.redrawer.MetalRedrawer
 import platform.UIKit.*
 import kotlin.system.getTimeNanos
 import org.jetbrains.skia.*
 
-actual open class SkiaLayer {
+@OptIn(InternalSkikoApi::class)
+actual open class SkiaLayer : SkiaLayerInterface {
+
+    @InternalSkikoApi
+    var needRedrawCallback: () -> Unit = {  }
+    @InternalSkikoApi
+    var detachCallback: () -> Unit = {}
```

Comment:
Should it be combined into some "java-style" listener?

Reply:
This class feels super redundant TBH, I think it's to be removed at all with proper refactoring.